sqlglot/parser.py

```diff
@@ -641,6 +641,7 @@ class Parser(metaclass=_Parser):
         TokenType.TIMESTAMP,
         TokenType.TIMESTAMPTZ,
         TokenType.TRUNCATE,
+        TokenType.UTC_DATE,
```

Comment:
Please add a test like `select utc_date` in `identity.sql`. I think this will fail to parse, in which case you should add this token to `ID_VAR_TOKENS`.

Reply:
Added a test. Looks like it succeeds.

Reply:
What does it parse into? It needs to be a `Column(Identifier("utc_date"))` and not a `UtcDate` expression.